src/envo/envo-edit.owl

```diff
@@ -28119,6 +28142,155 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000032> <http://purl.obolibrar
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000032> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_00000298>))
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000032> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002234> <http://purl.obolibrary.org/obo/ENVO_01000293>))
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000033> (marine sediment)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Pelagic_sediment#_note-8") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000033> "Sediment which accumulates within an ocean basin.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000033> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000033> "marine sediment"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000033> <http://purl.obolibrary.org/obo/ENVO_00002007>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000034> (shallow marine sediment)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Continental_shelf") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Pelagic_sediment#_note-8") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000034> "Marine sediment which accumulates within shallow regions of the oceanic basin close to continents, such as the continental shelf, or continental slope.")
```

Comment:
Isn't `occurs in` reserved for processes? Always check the domain and range.

You could say (for marine sediments in geneal) 
```
'output of' some ('sedimentation process' and 'occurs in' some 'marine water column')
```

Rephrase:

```
Marine sediment which has accumulated above the continental margins of an oceanic basin.
```

I've removed the "shallow" modifier and anchored to the features themselves. This won't be satisfactory for all. Please add an editor note saying that classes reflecting alternative definitions should be considered.

You don't need to add so much detail as the superclass contains it. You can copy the comment suggested in https://github.com/EnvironmentOntology/envo/pull/487#discussion_r109435466. 

Do we have continental margin? The union of the shelf, slope, and rise?